### ISSUE TITLE
request for styles/js returns html

### ISSUE THREAD

**Issue by ashconnell** [navgarcha/universal-react-saga#1]
Looks like Glue/hapi isn't correctly serving the public folder. 

I switched in expressjs and it worked but i'm unsure why hapi wasn't serving the static files correctly.

**Comment by navgarcha** [comment]
I've just done a fresh clone to check this and it's all working as expected. Ensure you've done a `npm run build` before you do `npm start`. Also make sure you do an `npm install` before all of this and check there are no errors during that stage.

**Comment by ashconnell** [comment]
That's not the problem. What node/npm version are you using? Unlikely the problem but worth checking quickly...

```
MacBook-Air:universal-react-saga ashconnell$ npm --version
3.10.3

MacBook-Air:universal-react-saga ashconnell$ node --version
v6.3.1

MacBook-Air:universal-react-saga ashconnell$ npm install
npm WARN universal-react-saga@1.0.0 No repository field.
npm WARN universal-react-saga@1.0.0 No license field.

MacBook-Air:universal-react-saga ashconnell$ npm run build

> universal-react-saga@1.0.0 build /Users/ashconnell/universal-react-saga
> webpack -d --config ./webpack/webpack.config.babel.js

clean-webpack-plugin: /Users/ashconnell/universal-react-saga/public/build has been removed.
Hash: 881bc75fdafc3196afa3
Version: webpack 1.13.1
Time: 4706ms
       Asset       Size  Chunks             Chunk Names
     main.js    1.27 MB       0  [emitted]  main
    main.css  180 bytes       0  [emitted]  main
 main.js.map    1.38 MB       0  [emitted]  main
main.css.map   85 bytes       0  [emitted]  main
    + 411 hidden modules
Child extract-text-webpack-plugin:
        + 2 hidden modules

MacBook-Air:universal-react-saga ashconnell$ npm start

> universal-react-saga@1.0.0 start /Users/ashconnell/universal-react-saga
> babel-node index.js

/Users/ashconnell/universal-react-saga/src/server
Server running at: http://localhost:8000
```

Note that the browser returns html in the main.js source.

![image](https://cloud.githubusercontent.com/assets/760516/17579992/20ab74ee-5fde-11e6-9873-2f39a8e2ad6c.png)

**Comment by navgarcha** [comment]
Just tried running with your exact same node version (6.3.1) and again i'm not having any issues. I'm guessing the JS file has HTML in it because its just a 404 response page. 

Is your localhost aliased to something odd? `cat /etc/hosts`

...although i guess that wouldn't explain why it works with express though.

**Comment by ashconnell** [comment]
Interestingly, i ensured it was still not working in my environment, cleared my cookies and refreshed, and it started working...

Unfortunately I didn't get a dump of my cookies and i'm unsure how they could have fixed it. Hapi must do something odd with cookies.

Regardless, i switched to using express because of the ecosystem and it made the code slightly cleaner.

**Comment by ashconnell** [comment]
I reproduced it by adding a cookie with a space in it's name. 

Here's the issue: https://github.com/hapijs/hapi/issues/2996

TLDR; Hapi was un-Hapi

**Comment by navgarcha** [comment]
I think i recall this issue when i first started using Hapi now you mention it! To be honest i'm from a more express background anyway, i just used Hapi here because this is broken off from a project i was doing a POC for. I might convert this over to express at some point.
